src/components/datasets/DatasetForm.js

```diff
@@ -9,26 +9,41 @@ import DataUseInput from "../DataUseInput";
 import { DATA_USE_PROP_TYPE_SHAPE, INITIAL_DATA_USE_VALUE } from "../../duo";
 import { simpleDeepCopy } from "../../utils/misc";
 
+const validateJson = (rule, value, callback) => {
+    try {
+        JSON.parse(value);
+        callback();
+    } catch (e) {
+        callback("Please enter valid JSON");
+    }
+};
+
 const DatasetForm = ({ style, initialValue, form }) => {
     return (
         <Form style={style || {}}>
             <Item label="Title">
                 {form.getFieldDecorator("title", {
-                    initialValue: (initialValue || { title: "" }).title || "",
+                    initialValue: initialValue?.title || "",
                     rules: [{ required: true }, { min: 3 }],
                 })(<Input placeholder="My Dataset" size="large" />)}
             </Item>
             <Item label="Description">
                 {form.getFieldDecorator("description", {
-                    initialValue: (initialValue || { description: "" }).description || "",
+                    initialValue: initialValue?.description || "",
                     rules: [{ required: true }],
                 })(<Input.TextArea placeholder="This is a dataset" />)}
             </Item>
             <Item label="Contact Information">
                 {form.getFieldDecorator("contact_info", {
-                    initialValue: (initialValue || { contact_info: "" }).contact_info || "",
+                    initialValue: initialValue?.contact_info || "",
                 })(<Input.TextArea placeholder={"Name\nInfo@c3g.ca"} />)}
             </Item>
+            <Item label="DATS File">
+                {form.getFieldDecorator("dats_File", {
```

Comment:
are you sure about the casing? `dats_File`? I would have expected `dats_file` as in the propTypes